salsa-2022-tests/tests/compile-fail/specify-does-not-work-if-the-key-is-a-salsa-input.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0277]: the trait bound `MyInput: TrackedStructInDb<dyn Db>` is not satisfied
+   --> tests/compile-fail/specify-does-not-work-if-the-key-is-a-salsa-input.rs:21:28
+    |
+20  | #[salsa::tracked(jar = Jar, specify)]
+    | ------------------------------------- required by a bound introduced by this call
+21  | fn tracked_fn(db: &dyn Db, input: MyInput) -> MyTracked {
+    |                            ^^^^^ the trait `TrackedStructInDb<dyn Db>` is not implemented for `MyInput`
```

Comment:
I wonder if there is a way to make the error message more friendly. Maybe something like "specify only works if the key is the tracked struct", but get a `salsa::input` here.